howso/direct/client.py

```diff
@@ -1160,16 +1132,6 @@ def _auto_resolve_trainee(self, trainee_id: str):
         """
         if trainee_id not in self.trainee_cache:
             self.acquire_trainee_resources(trainee_id)
```

Comment:
Seems to be that previously, the entity_id was helping us effectively create a sort of re-entrant lock around a trainee so that it would not be abused by writes from multiple processes. Is it OK to do this now? Is there some other mechanism to prevent this? Genuine questions.

Reply:
The entity_id before seemed to be used to check that the Trainee that was to be interacted with actually was contained in the Howso Entity that was being controlled by the client. 

This was necessary since trainees could be in separate Howso entities loaded within the same Amalgam instance. No longer necessary since there are no Howso entities.